assets/models/system/microsoft-orca-2-7b/asset.yaml

```diff
@@ -0,0 +1,4 @@
+extra_config: model.yaml
+spec: spec.yaml
+type: model
+categories: ["Models", "Text Generation"]
```

Comment:
(nit) add new line at the end of the file?

Reply:
Done